### PR TITLE
Add Glossary Terms

### PR DESCRIPTION
**Pull request by ciaranightingale**

### CODE REVIEW

File: docs/onboarding/15 Glossary/Claiming.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+slug: /glossary/claiming
+title: Claiming
+---
+
+Claiming is the process of transferring token ownership to the claimer's wallet.
+Once a token has been [minted](/glossary/minting) or [lazy minted](/glossary/lazy-minted), it can be claimed by a wallet.
```
Comment (nachoiacovino) [inline docs/onboarding/15 Glossary/Claiming.mdx:7]: if the token is minted, there's no claiming process, only if it's lazy minted

File: docs/onboarding/15 Glossary/Composability.MDX [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+slug: /glossary/composability
+title: Composability
+---
+
+Composability refers to the ability for different protocols and smart contracts to interact and build upon each other. 
+It allows for the creation of new products and services without having to build everything from scratch. 
+
+Thirdweb is completely composible. Each product can be used independently or in combination with other products.
```
Comment (nachoiacovino) [inline docs/onboarding/15 Glossary/Composability.MDX:9]: capitalization hurts my soul

File: docs/onboarding/15 Glossary/Composability.MDX [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+For example, you can use the [SDKs](/sdk) or [Dashabord](/dashboard) with **any** contract. Similarly, you can use the [Solidity SDK](/solidity)
```
Comment (nachoiacovino) [inline docs/onboarding/15 Glossary/Composability.MDX:10]:
```suggestion
For example, you can use the [SDKs](/sdk) or [Dashboard](/dashboard) with **any** contract. Similarly, you can use the [Solidity SDK](/solidity) 
```

File: docs/onboarding/15 Glossary/Factory Contract.mdx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+slug: /glossary/factory-contract
+title: Factory Contract
+---
+
+A factory contract is a smart contract that is used to deploy other smart contracts.
```
Comment (nachoiacovino) [inline docs/onboarding/15 Glossary/Factory Contract.mdx:6]: I feel like we could add much more here talking about we leverage these to have 10x gas savings etc and talk about proxy contracts?

File: docs/onboarding/15 Glossary/NFT.mdx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+slug: /glossary/nft
+title: NFT
+---
+
+A Non-fungible token (NFT) is a type of cryptographic token that represents something unique;
+non-fungible tokens are thus not mutually interchangeable. 
+It is defined by the [ERC-721 standard](https://eips.ethereum.org/EIPS/eip-721), which is a standard for creating NFTs on the Ethereum blockchain.
```
Comment (nachoiacovino) [inline docs/onboarding/15 Glossary/NFT.mdx:8]: are we not talking about 1155s here? they are also considered NFTs

File: docs/onboarding/15 Glossary/Permissionless.mdx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+slug: /glossary/permissionless
+title: Permissionless
+---
+
+Nn the context of smart contracts, permissionless means that anyone can interact with the contract without needing special permission or approval.
```
Comment (nachoiacovino) [inline docs/onboarding/15 Glossary/Permissionless.mdx:6]:
```suggestion
On the context of smart contracts, permissionless means that anyone can interact with the contract without needing special permission or approval. 
```

File: docs/onboarding/15 Glossary/Claiming.mdx (hunk 1 above)
Comment (ciaranightingale) [reply docs/onboarding/15 Glossary/Claiming.mdx:7]: good catch

File: docs/onboarding/15 Glossary/Composability.MDX (hunk 2 above)
Comment (ciaranightingale) [reply docs/onboarding/15 Glossary/Composability.MDX:9]: start of sentence brpo

File: docs/onboarding/15 Glossary/NFT.mdx (hunk 5 above)
Comment (ciaranightingale) [reply docs/onboarding/15 Glossary/NFT.mdx:8]: true but they are semi-fungible - can add something tbf

**Pull request merged**